controllers/changetierrequest/changetierrequest_controller_test.go

```diff
@@ -53,6 +53,7 @@ func TestChangeTierSuccess(t *testing.T) {
 		// then
 		require.NoError(t, err)
 		murtest.AssertThatMasterUserRecord(t, "john", cl).
+			HasTier(*teamTier).
```

Comment:
it's just a detail, but we don't have to pass the whole `NSTemplateTier` instance - just the name is enough

Reply:
the `HasTier` func is in toolchain-common. We can change that later if it causes troubles.